### PR TITLE
app.config.main.paths.www -> for Windows FIX

### PR DESCRIPTION
**Pull request by imkarma**
On windows was error "Invalid path". Becouse plugin take window seperator "\". Now its fine :)

### CODE REVIEW

**Comment by konstantinzolotarev** [comment]
@imkarma not really sure that it's fixed. because `staticDir` might contain Windows path separator.

**Comment by imkarma** [comment]
@konstantinzolotarev now I use "trailpack-hapi": "git+https://github.com/imkarma/trailpack-hapi.git" and its work fine. Becouse if use old code I have

``` javascript
server.route({
        method: 'GET',
        path: '\static\{filename*}',
        ....
      })
```

but right syntaxis 

``` javascript
server.route({
        method: 'GET',
        path: '/static/{filename*}',
        ....
      })
```

it's becouse Windows dir seperator '\' but we must have '/' like on unix os

File: lib/server.js [hunk 1]
```diff
@@ -37,7 +37,7 @@ module.exports = {
       const staticDir = path.relative(app.config.main.paths.root, app.config.main.paths.www)
       server.route({
         method: 'GET',
-        path: path.join(path.sep, staticDir, '{filename*}'),
```
Comment (scott-wyatt) [inline lib/server.js:40]: What about `path.normalize(path.join(path.sep, staticDir, '{filename*}'))` here or something?

**Comment by imkarma** [comment]
@scott-wyatt 
problem not with path. Promlem convert windowc path to string to applay http path. on windows we have \static\{filename_}, with path.normalize we will have somwthin like this \\static\\{filename_}, but we need /static/{filename*} for http requests http://example.com/static/index.html for example

**Comment by konstantinzolotarev** [comment]
@imkarma Yep I know. I mean this code: 
`const staticDir = path.relative(app.config.main.paths.root, app.config.main.paths.www)`
Could contain path `something\some` if 

```
app.config.main.paths.root = 'something'
app.config.main.paths.www = 'some'
```

And your fix wouldn't work because it will include wrong `staticDir`

**Comment by imkarma** [comment]
@konstantinzolotarev Now i understend what you mean and add fix, I hope :)

**Comment by imkarma** [comment]
@konstantinzolotarev sorry can you look to this pull request again I add RegEx and now it must be cool :) Or geve feedback and i try rewritte if needed

**Pull request merged**